acct-user/bitcoin/bitcoin-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="System-wide Bitcoin services user"
+ACCT_USER_ID=483
+ACCT_USER_HOME=/var/lib/bitcoin
```

Comment:
Does this user really need a home directory? If it can not have one, that would be ideal.

Reply:
I'm not sure. It's always had one. Current init scripts set `-datadir` explicitly, but it seems probable that older ones might not (on the other hand, maybe we don't care?).

(The directory is certainly needed to store data, so I assume you're only asking about it being `$HOME`)

Reply:
> (The directory is certainly needed to store data, so I assume you're only asking about it being `$HOME`)

Yep :)

Can you please do a test and find out? If we can not have it be `$HOME`, that would be ideal.

Reply:
They all use the same init script, so it should be good to go.